teos/cli/teos_cli.py

```diff
@@ -14,37 +15,65 @@
 from common.exceptions import InvalidKey, InvalidParameter, SignatureError, TowerResponseError
 
 from teos import DEFAULT_CONF, DATA_DIR, CONF_FILE_NAME
-from teos.cli.help import show_usage, help_get_all_appointments
+from teos.cli.help import (
+    show_usage,
+    help_get_all_appointments,
+    help_get_tower_info,
+    help_get_users,
+    help_get_user,
+)
 from teos.protobuf.tower_services_pb2_grpc import TowerServicesStub
+from teos.protobuf.user_pb2 import GetUserRequest
 
 
-def get_all_appointments(rpc_host, rpc_port):
+def to_json(obj):
+    return json.dumps(obj, indent=4)
+
+
+def formatted(func):
+    """Transforms the given function by wrapping the return value with json_format.MessageToDict followed by
+    json.dumps, in order to print the result in a prettyfied json format.
     """
-    Gets information about all appointments stored in the tower.
 
-    Args:
-        rpc_host (:obj:`str`): the hostname (or IP) where the rpc server is running.
-        rpc_port (:obj:`int`): the port where the rpc server is running.
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs):
+        result = func(*args, **kwargs)
+        result_dict = json_format.MessageToDict(
+            result, including_default_value_fields=True, preserving_proto_field_name=True
+        )
+        return to_json(result_dict)
+
+    return wrapper
+
 
-    Returns:
-        :obj:`dict` a dictionary containing all the appointments stored by the Responder and Watcher if the tower
-        responds.
+class RPCClient:
+    """Creates and keeps a connection to the an RPC serving TowerServices. It has methods to call each of the
+    available grpc services, and it returns a pretty-printed json response.
```

Comment:
Arguments and Attributes are missing in the docs.

Reply:
Added in 7d75df813325e0ce08717101ac10c3ec23e43862.